Source/JavaScriptCore/bytecode/StructureStubInfo.h

```diff
@@ -268,6 +278,9 @@ class StructureStubInfo {
             }
             
             bufferingCountdown--;
+
```

Comment:
So what is an example of a non-cell that the megamorphic cache would want to cache? A property miss on null?

Reply:
`considerCachingMegamorphic` is used for megamoprhic cache's miss. Non cell case, we eventually go to generic case.